builder/vmware/iso/builder_acc_test.go

```diff
@@ -15,8 +18,25 @@ func TestBuilderAcc_basic(t *testing.T) {
 		t.Fatalf("failed to load template file %s", templatePath)
 	}
 
-	builderT.Test(t, builderT.TestCase{
-		Builder:  &Builder{},
+	testCase := &acctest.PluginTestCase{
```

Comment:
example of test after being upgraded to the generic framework.

Reply:
nice.